core/state/history_test.go

```diff
@@ -20,7 +20,7 @@ func TestNewStateHistory(t *testing.T) {
 		assert.NotNil(t, history.state)
 	})
 
-	t.Run("invalid state root", func(t *testing.T) {
+	t.Run("bigger state root", func(t *testing.T) {
```

Comment:
Can we revert this change?

Reply:
looks like mistake on my side during rebase